lib/booktype/urls.py

```diff
@@ -30,6 +30,9 @@
 )
 
 urlpatterns = [
+    # django log tailer
+    url(r'^logs/', include('logtailer.urls')),
```

Comment:
aha! @aerkalov I see you're looking for fun uh? we have couple of interesting tickets if you would like to have some real fun ;) haha